packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -39,27 +39,13 @@ const indeterminatePath: IconAsset = {
 /**
  * The internal stateless ☑️ Checkbox
  */
-export default class CheckboxCore extends React.Component<ChoiceCoreProps> {
-    componentDidMount(): void {
-        if (this.props.checked == null && this.inputRef.current != null) {
-            this.inputRef.current.indeterminate = true;
```

Comment:
I'm a little confused here... removing this `componentDidMount()` and `componentDidUpdate()` didn't seem to change anything. All the behavior works as expected. And I don't think we're using the internal value of `indeterminate` anywhere?

Can I keep it this way without setting this value? Or would it be preferable for me to add this back in a `useEffect()`? (cc @kevinbarabash @jandrade)

Reply:
Is this so that the `:indeterminate` pseudoselector can be used? I don't think we use pseudoselectors in our code though..

Reply:
This is to programmatically add the `indeterminate` value. You can see an explanation here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes

> This is set using the [HTMLInputElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement) object's indeterminate property via JavaScript (it cannot be set using an HTML attribute)